tests/braille-specs/en-ueb-computer-code.yaml

```diff
@@ -104,40 +118,48 @@ tests:
 - - 'context accept'
   - '3text a3ept'
 - - ':text a:ept'
-  - ';:text a;:ept'
-  - {xfail: 'grade 1 indicator is missing preceding ":"'}
+  - ';3text a;3ept'
```

Comment:
So the original test was wrong?

Reply:
Yep, they used `:` instead of `3` for dots 25